### DISCUSSION TITLE
How to set DateTime field min value to now?

### DISCUSSION THREAD

**Discussion by yatochka-dev** [pocketbase/pocketbase#1971] [Q&A] [answered]
I have `due` field in my todos row, I want to filter creating of each todo, that it won't be possible to create a todo with due date in the past. 

I've tried this API rule: 
![image](https://user-images.githubusercontent.com/93011550/222927831-72092062-cb9a-46cf-852a-4c3eefc2ccdb.png)

but it didn't work, any suggestions/ideas how I can implement this?

**Comment by ganigeorgiev** [comment, marked as answer]
If I understand your use case, reversing the rule should achieve what you are looking for, eg.:

```js
@request.data.due > @now
```
The above roughly translates to: _"Require the submitted `due` datetime to be greater than the current datetime (aka. needs to be a date in the future)."_
